stdlib/public/TensorFlow/Ops.swift

```diff
@@ -1600,25 +1600,36 @@ public extension Tensor {
 
 public extension Tensor {
   @inlinable
+  // SWIFT_ENABLE_TENSORFLOW
+  @differentiable(wrt: self, vjp: _vjpBroadcast(toShape:)
+    where Scalar : TensorFlowFloatingPoint)
   func broadcast(toShape shape: Tensor<Int32>) -> Tensor {
     return Raw.broadcastTo(self, shape: shape)
   }
 
   @inlinable
+  // SWIFT_ENABLE_TENSORFLOW
```

Comment:
No need for `// SWIFT_ENABLE_TENSORFLOW` if the code base you are working in is already different from the master branch. In this case, the file is already a TensorFlow-specific file.

Reply:
Ahh yes I realized the other Gradients file was all tensorflow specific, but completely forgot this is as well!